ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKTableTest.java

```diff
@@ -253,4 +273,136 @@ public void shouldGroupKeysCorrectly() {
     assertThat(keyValue.key, equalTo("bar|+|foo"));
     assertThat(keyValue.value, equalTo(value));
   }
+
+
+  @SuppressWarnings("unchecked")
+  @Test
+  public void shouldPerformTableToTableLeftJoin() {
+
+    final KTable mockKTable= EasyMock.niceMock(KTable.class);
+    expect(mockKTable.leftJoin(anyObject(KTable.class),
+                            anyObject(SchemaKStream.KsqlValueJoiner.class)))
+        .andReturn(EasyMock.niceMock(KTable.class));
+
+    replay(mockKTable);
+    initialSchemaKTable= new SchemaKTable(ksqlTable.getSchema(), mockKTable,
```

Comment:
Same comments as in the `SchemaKStreamTest` - can extract the common code in the tests to methods etc.